### PR TITLE
virttest.qemu_monitor: fix sytax error when pick monitor from env

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

File: virttest/qemu_monitor.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def __setstate__(self, state):
     def __getinitargs__(self):
         # Save some information when pickling -- will be passed to the
         # constructor upon unpickling
-        return self.vm, self.name, self.filename
+        return self.vm, self.name, self.filename, True
```
Comment (PandaWei) [inline virttest/qemu_monitor.py]: You set the default of suppress_exception to False.
but the getinitargs return True.
Is it OK? I'm not sure ... Just looks odd.

Comment (xutian) [inline virttest/qemu_monitor.py]: yes, it looks really a little odd. let me update it. thanks

**Comment by xutian** [comment]
@PandaWei  updated, thanks.

**Comment by PandaWei** [comment]
ACK

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by xutian** [comment]
thanks @PandaWei and @CongLi , merged.

**Pull request merged**